src/components/views/settings/tabs/user/GeneralUserSettingsTab.tsx

```diff
@@ -454,17 +454,17 @@ export default class GeneralUserSettingsTab extends React.Component<IProps, ISta
         const msisdns = this.state.loading3pids ? <Spinner /> : <DiscoveryPhoneNumbers msisdns={this.state.msisdns} />;
 
         const threepidSection = this.state.haveIdServer ? (
-            <div className="mx_GeneralUserSettingsTab_discovery">
+            <>
                 <span className="mx_SettingsTab_subheading">{_t("Email addresses")}</span>
                 {emails}
 
                 <span className="mx_SettingsTab_subheading">{_t("Phone numbers")}</span>
                 {msisdns}
-            </div>
+            </>
         ) : null;
 
         return (
-            <div className="mx_SettingsTab_section">
+            <div className="mx_SettingsTab_section mx_GeneralUserSettingsTab_section--discovery">
```

Comment:
This should not change how `.mx_SettingsTab_subheading` and `.mx_Spinner` are displayed, as both of them are elements inside `threepidSection`, which this element consists.